spec/ddtrace/contrib/rest_client/patcher_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'spec_helper'
+require 'ddtrace'
+
+RSpec.describe Datadog::Contrib::RestClient::Patcher do
+  describe '.patch' do
+    let!(:rest_client_request_class) { class_double('RestClient::Request').as_stubbed_const }
+
+    before do
+      described_class.send(:unpatch)
+    end
+
+    context 'when delayed job is not present' do
+      before do
+        hide_const('RestClient::Request')
+      end
+
+      it "shouldn't patch the code" do
```

Comment:
Minor, non-blocking English thing, but RSpec discourages the use of the word "should"; they would normally suggest this be written as `it "does not patch the code"` or `it "patches the code"`, in the assertive form.

Reply:
Sounds better! Thanks for suggestion